test/processors/text_processor_test.rb

```diff
@@ -1,7 +1,15 @@
 require 'test_helper'
 
 class TextProcessorTest < ActiveSupport::TestCase
-  let(:processor) { TextProcessor.new(logger: Logger.new('/dev/null')) }
+  let(:audio_monster) do
+    Minitest::Mock.new
+  end
+
+  let(:processor) do
+    TextProcessor.new(logger: Logger.new('/dev/null')).tap do |p|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.